# Representation of the world state in the SML World

VehicleState[] vehicle_states
TrafficDemand[] traffic_demand
int32[] bus_stop_ids
int32[] bus_stop_demands
int32 time
